ckan/templates/group/about.html

```diff
@@ -3,14 +3,22 @@
 {% block subtitle %}{{ _('About') }} - {{ super() }}{% endblock %}
 
 {% block primary_content_inner %}
-  <h1>{% block page_heading %}{{ c.group_dict.display_name }}{% endblock %}</h1>
+  <h1>
+    {% block page_heading %}
+      {% if c.group_dict.title %}
```

Comment:
True, had the same idea before but forgot to actually do the fix. I'll commit the same fix to the other remaining templates in a moment.